src/cli/cli.ts

```diff
@@ -1,45 +1,75 @@
-import { getLanguages, LanguageCodes, Sources } from '..';
+import { getLanguages,  Sources } from '..';
 import { fileTranslator, getFileFromPath } from '../core/json_file';
 import {
-  commands,
   error,
   info,
-  language_choices,
   messages,
   success,
   warn,
 } from '../utils/console';
 import loading from 'loading-cli';
-import { getCodeFromLanguage, translationStatistic } from '../utils/micro';
+import { capitalize, current_version, getCodeFromLanguage, translationStatistic } from '../utils/micro';
 import { readProxyFile } from '../core/proxy_file';
-var inquirer = require('inquirer');
+import { Command } from "commander";
+import { promptFrom, promptTo, promptTranslator } from '../utils/prompt';
+var figlet = require("figlet");
+
+const program = new Command();
 
 export async function initializeCli() {
   global.totalTranslation = 0;
   global.totalTranslated = 0;
   global.proxyIndex = 0;
   global.proxyList = [];
+  program
+    .version(current_version)
+    .usage(messages.cli.usage)
+    .description(`${messages.cli.usageWithProxy}\n${messages.cli.usageByOps}\n\n\n${messages.cli.description}`)
+    .option("-T, --translator <TranslationService>", messages.cli.translator, false)
```

Comment:
```suggestion
    .addOption(
      new Option("-T, --translator <TranslationService>", messages.cli.translator)
      .choices(Object.values(Sources))
    )
```

Doing this way we have 2 benefits:
1. User will see which options they can to use;
2. We make sure that our available options always synced with `enum Sources` (yes we would need to get rid of `switch..case` below.)

Reply:
Excuse me .  Busy these days, I will work on that in the near future

Reply:
@Demetri0 @FathiGuemri  Totally agree with the short name. I guess that would be much easier for user to remember and type.

What also we can add is, if user types nonsense language we can show the link of the [LANGUAGES.md](https://github.com/mololab/json-translator/blob/master/docs/LANGUAGES.md) for checking short forms of languages. But ofc first need to update [LANGUAGES.md](https://github.com/mololab/json-translator/blob/master/docs/LANGUAGES.md) to store short names.

PS: also I am extremely busy nowadays, love & plan to help more when I got free times.

Thanks guys, you are brilliant 💜

Reply:
@ParvinEyvazov I did the required. But I have to check **LANGUAGES.md** because I'm relying on ChatGPT